docs/src/gui/qtvcp-vcp-panels.adoc

```diff
@@ -10,192 +10,198 @@
+- The text box and state can be output as speech if sound is selected.
+- A `combobox` can be used to automatically select and connect to a
+  pin/signal.
+- You can add more LEDs from the drop down menu.
+- The LED can be detached from the main windows.
 
-Here is how to load test_led from a HAL script. +
-The '-o' switch sets how many LEDs the panel starts with. +
-(If loading directly from a terminal omit the 'loadusr')
+Here is how to load `test_led` from a HAL script:
 
 [source,{hal}]
 ----
 loadusr qtvcp test_led
 loadusr qtvcp -o 4 test_led
 ----
 
-.test dial - Test Dial VCP
-image::images/qtvcp_test_led.png["Test LED",align="left"]
+The `-o` switch sets how many LEDs the panel starts with. +
+If loading directly from a terminal omit the 'loadusr'.
 
-=== test_panel
+.QtVCP `test_dial` Panel - Test LED VCP
+image::images/qtvcp_test_led.png["QtVCP test_dial Panel - Test LED VCP",align="center"]
 
-- collect of useful widgets for testing HAL component. Including speech
-  of LED state.
+[[sub:qtvcp:panels:test-panel]]
+=== `test_panel`
+
+*Collection of useful widgets for testing HAL component*, including
+speech of LED state.
 
 [source,{hal}]
 ----
 loadusr qtvcp test_panel
 ----
 
-.test panel - Test Panel VCP
-image::images/test_panel.png["Test Panel",align="left"]
+.QtVCP `test_panel` - HAL Component Testing Panel
+image::images/test_panel.png["QtVCP test_panel - HAL Component Testing Panel",align="center"]
 
-=== cam_align
+[[sub:qtvcp:panels:cam-align]]
+=== `cam_align`
 
-- a camera display widget for rotational alignment
+A *camera display widget for rotational alignment*.
 
-.QtVCP cam align- Camera based alignment panel
-image::images/qtvcp-cam-align.png["Qt Camera Alignment",align="left"]
+.QtVCP `cam_align` Panel - Camera Based Alignment Panel
+image::images/qtvcp-cam-align.png["QtVCP cam_align Panel - Camera Based Alignment Panel",align="center"]
 
-=== sim_panel
+[[sub:qtvcp:panels:sim-panel]]
+=== `sim_panel`
 
-- a small control panel to simulate MPG jogging controls etc. for
-  simulated configurations
+Small control panel to *simulate MPG jogging controls etc*. +
+For simulated configurations
 
 [source,{hal}]
 ----
 loadusr qtvcp sim_panel
 ----
 
-.QtVCP sim panel - Simulated controls for screen testing.
-image::images/qtvcp_sim_panel.png["Sim Panel",align="left"]
-
-[[cha:qtvcp-vismach-panels]]
+.QtVCP `sim_panel` - Simulated Controls Panel For Screen Testing.
+image::images/qtvcp_sim_panel.png["QtVCP sim_panel - Simulated Controls Panel For Screen Testing",align="center"]
 
-=== vismach_mill_xyz
+[[sub:qtvcp:panels:vismach]]
+=== `vismach_*` Panels
 
-- 3d openGL view of a 3 axis milling machine
+.`vismach_mill_xyz`
+3D OpenGL view of a _3-Axis milling machine_.
 
 [source,{hal}]
 ----
 loadusr qtvcp vismach_mill_xyz
 ----
 
-.QtVCP vismach- 3 Axis Mill panel
-image::images/qtvismach.png["QtVCP Vismach Mill",align="left"]
+.QtVCP `vismach_mill_xyz` - 3-Axis Mill 3D View Panel
```

Comment:
```suggestion
=== QtVCP `vismach_mill_xyz` - 3-Axis Mill 3D View Panel
```

and so on...

Reply:
Go for it, will just keep the titles short.